pkg/cmd/pr/merge/merge.go

```diff
@@ -365,6 +377,25 @@ func mergeRun(opts *MergeOptions) error {
 	return nil
 }
 
+func PullLatestChanges(opts *MergeOptions, repo ghrepo.Interface, branchToSwitchTo string) error {
```

Comment:
Nit: the argument name `branchToSwitchTo` could just be called `branch`. It's implied by the function name that this branch will be pulled to, not switched to.